docs/pages/server-access/guides/vscode.mdx

```diff
@@ -53,17 +74,15 @@ in the path below:
   </TabItem>
 </Tabs>
 
-You should be able to connect to the desired node using following command:
+You should be able to connect to the desired Node using following command:
 
 ```code
-ssh user@[node name].[cluster name]
+$ ssh user@[node name].[cluster name]
```

Comment:
Probably my fault, but I'm having trouble with this part.  I have a cloud account.  I have an email address for a username.  Not quite sure how to format the username, node name, and cluster name in this command.  If I do it exactly as specified above, I get a permission denied error.

Also, I'm not sure why the word "node" is capitalized.

Reply:
It looks like openssh parses the username by finding the first instance of `@` in the URL and terminating the string at it (https://github.com/openssh/openssh-portable/blob/8e206e0dd6b9f757b07979e48f53ad5bf9b7b52b/misc.c#L994-L997), so a username with a `@` in it won't parse correctly. @timothyb89 Since you wrote the original guide, do you know what the correct course of action is here? Otherwise, I'm happy to set up a lab and try this out.

Reply:
Got it to work.  First, I figured out that to login with an email address, you can enclose the email address in single quotes ('email@example.com').  (This wasn't tested though.).  Second, I realized that you're supposed to be using the username _on the node_, not the username in Teleport.  Dur.  

So these instructions are fine.  I tested them successfully after running tsh login.  Example command: 
`ssh username-on-node@ip-10-0-1-161.us-west-2.compute.internal.paul.cloud.gravitational.io`

Reply:
Yup, that's exactly correct. I _think_ we behave the same as `tsh ssh` here but perhaps not.

Reply:
Thanks for testing this out! I've clarified the value of `user` in the `ssh` command.